CHANGELOG.md

```diff
@@ -5,9 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
-- [Extensions] Moving Extensions APIs to support cross versions via protobuf. ([#6960](https://github.com/opensearch-project/OpenSearch/pull/6960))
+- [Extensions] Moving Extensions APIs to support cross versions via protobuf. ([#7402](https://github.com/opensearch-project/OpenSearch/issues/7402))
```

Comment:
The changelog looks a bit messed up

Reply:
Updated this as per the merge on #7402
Updated the main changelog on #7409

Reply:
My bad @reta. I've started merging in more PRs but it didn't make sense for readers as all of them are just moving these APIs to support cross versions. So I've just moved it to meta issue and updated main and 2.x.